src/app/modules/auth/auth.component.html

```diff
@@ -10,10 +10,15 @@
 </nav>
 <router-outlet></router-outlet>
 <app-pop-up popUpName="shopping" [title]="popUpTitle" [open]="popUp" (closeAction)="closePopUp($event)">
-  <ul>
-    <li *ngFor="let book of shoppingBooks; index as i" class="list-item">
+  <ul class="list-items" *ngIf="shoppingBooks.length > 0;else emptyStore">
```

Comment:
ngIf checks the boolean value of the expression. Therefore, `shoppingBooks.length` is enough.